### ISSUE TITLE
Railroader blocks Ruby Parser

### ISSUE THREAD

**Issue by mensfeld** [david-a-wheeler/railroader#5]
Is there a chance to bump up Ruby Parser?

I cannot use the Railroader in Coditsu because it uses an old Ruby parser:

```
Bundler could not find compatible versions for gem "ruby_parser":
  In Gemfile:
    offending_engine was resolved to 4.37.0, which depends on
      fasterer (= 0.5.1) was resolved to 0.5.1, which depends on
        ruby_parser (>= 3.13.0)

    offending_engine was resolved to 4.37.0, which depends on
      railroader was resolved to 4.3.4, which depends on
        ruby_parser (~> 3.11.0)
```

**Comment by david-a-wheeler** [comment]
I won't know how hard it is until I try, but I suspect it will be no big deal. We shall see.

**Comment by mensfeld** [comment]
Thank you. Literally the moment it's up I will make an updated release to Coditsu as all (seriously all) things work well!

**Comment by mensfeld** [comment]
Hello @david-a-wheeler no need to rush anymore. I've managed to bypass that obstacle by isolating railroader and it should hit production after the weekend.

**Comment by david-a-wheeler** [comment]
Fantastic!

**Comment by david-a-wheeler** [comment]
I've updated ruby_parser to 3.12 within Railroader.

I could easily update ruby_parser to the current 3.13 within Railroader, but the side-effect of doing that is that Railroader cannot handle Ruby 1.9.  I'm sure I'll do that eventually, the question is "when".  I suspect I'll do that soon, but I want to give people a chance to comment about that. For more about that, see issue https://github.com/david-a-wheeler/railroader/issues/6

**Comment by david-a-wheeler** [comment]
I'm closing this issue.  You've resolved it, and issue #6 captures the "update ruby_parser at some point" with more info.
